cvat/apps/lambda_manager/views.py

```diff
@@ -621,6 +621,9 @@ def reset(self):
                         "source": "auto"
                     }
 
+                    if anno.get("rotation"):
+                        shape["rotation"] = anno.get("rotation")
```

Comment:
``rotation`` is only supported actually for ellipses and bounding boxes. 
So, we need additional check for shape type here I believe, otherwise it can lead to unexpected results later. 

```python
if shape["type"] in ("rectangle", "ellipse"):
    shape["rotation"] = anno.get("rotation", 0)
```